content/blog/2018-08-09-prometheus-graduates-within-cncf.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Prometheus Graduates Within CNCF
+created_at: 2018-08-09
+kind: article
+author_name: Richard Hartmann
+---
+
+We are happy to announce that as of today, Prometheus graduates within CNCF.
+
+Prometheus is the second project ever to make it to this tier.
+By graduating Prometheus, CNCF shows that it's confident in our code and feature velocity, our maturity and stability, and our governance and community processes.
+This also acts as an external verification of quality for anyone needing this for their internal Q&A.
+
+Since reaching incubation level, a lot of things happened; some of which stand out:
+
+* We redid our storage back-end to support high churn in services
+* We had a large push towards stability, with 2.3.2 having no known bugs or issues
```

Comment:
Uuuhm, SoundCloud still needs to regularly downgrade servers due to service discovery getting stuck. There are more reports from others. Sounds like a very bold claim.

Reply:
2.3.2 has one known bug